go/test/endtoend/cluster/cluster_process.go

```diff
@@ -137,6 +147,20 @@ func (shard *Shard) Replica() *Vttablet {
 	return nil
 }
 
+// CtrlCHandler handles the teardown for the ctrl-c.
+func (cluster *LocalProcessCluster) CtrlCHandler() {
+	cluster.Context, cluster.CancelFunc = context.WithCancel(context.Background())
+
+	c := make(chan os.Signal, 2)
```

Comment:
why is the channel buffer of size 2? Can you add the explanation as a comment in the code?

Reply:
generally people uses in that way to handle the blocking in signal notifier.